app/build.gradle

```diff
@@ -71,6 +71,10 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8
```

Comment:
What do we need this for integration tests, please clarify?

Reply:
The app build was failing due to using the `androidx.test.ext.junit.runners.AndroidJUnit4`,  the ` androidx.test.runner.AndroidJUnit4` also works but it has been deprecated.